template.json

```diff
@@ -75,6 +72,7 @@
     "stylelint-processor-styled-components": "^1.6.0",
     "ua-parser-js": "^0.7.19",
     "@reduxjs/toolkit": "^1.2.1",
-    "immer": "^5.1.0"
+    "immer": "^5.1.0",
+    "react-test-renderer": "^16.12.0"
   }
```

Comment:
I would add [babel-plugin-react-remove-properties](https://www.npmjs.com/package/babel-plugin-react-remove-properties)  to remove `data-testid` from elements

Reply:
It seems like we can't add any custom babel plugin inside CRA template - this can be only done after ejecting the created project if needed.

Anyway, in this [thread](https://github.com/testing-library/react-testing-library/issues/479) the creator of the library suggests leaving those props in the code but treating them as a last-resort query option (use other queries whenever possible).

That being said, I've changed the test to use better selector.